onyo/commands/tests/test_mkdir.py

```diff
@@ -153,25 +153,6 @@ def test_dir_exists(repo: OnyoRepo, directory: str) -> None:
     assert repo.git.is_clean_worktree()
 
 
-@pytest.mark.repo_files(*directories)  # used as files to test errors
-@pytest.mark.parametrize('file', directories)
-def test_dir_exists_as_file(repo: OnyoRepo, file: str) -> None:
```

Comment:
I think this test (and similar CLI tests later) should not be deleted.

Instead, the assertions just need minimal adjustments, e.g. not erroring anymore, and e.g.
`assert Path(file).is_file()` -> `assert Path(file).is_directory()`.

Reply:
This should be tested in the python tests, IIRC.
I'd like to not repeat functionality tests in the CLI tests, but rather just test the wrapper does the correct wrapping :)